src/components/extensive/WorkdayCollapseGrid/hooks.ts

```diff
@@ -19,29 +30,29 @@ export function calculateTotals(demographics: Demographic[]) {
       counts[type] += amount;
       return counts;
     },
-    { gender: 0, age: 0, ethnicity: 0 }
+    { gender: 0, age: 0, ethnicity: 0, caste: 0 }
   );
 
-  const total = Math.max(counts.age, counts.gender, counts.ethnicity);
-  const countsMatch = uniq([counts.age, counts.gender, counts.ethnicity]).length === 1;
-  return { total, countsMatch };
+  const total = Math.max(counts.age, counts.gender, counts.ethnicity, counts.caste);
+  const countsMatch = uniq([counts.age, counts.gender, counts.ethnicity, counts.caste]).length === 1;
+  return { counts, total, countsMatch };
```

Comment:
This is going to break for the existing usage of the workday table. Since non-HBF uses don't have `caste`, it's always going to be 0, and will therefore show as "not balanced" in the editor because `countsMatch` will always be false. Once you have a new strategy in place here, please test with non-HBF frameworks to make sure the existing functionality isn't impacted.